src/cockpit/protocol.py

```diff
@@ -237,7 +237,8 @@ async def request_authorization(
     ) -> str:
         if self.authorizations is None:
             self.authorizations = {}
-        cookie = str(uuid.uuid4())
+        cookie = str(self.next_auth_id)
```

Comment:
I agree -- this is just for telling multiple parallel auth requests apart, it's not  a secret.